docs/manual/config/index.rst

```diff
@@ -234,3 +233,5 @@ session manager and use gnome as usual.
 
     without-dm
     gnome
+
+
```

Comment:
Looks like you left in some empty lines

Reply:
Ha. Wha? That was too quick!

Reply:
I was curious to see if the PR was indeed resolving what was discussed in ORC earlier, and it is :)